feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/staking/controller/SetControllerViewModel.kt

```diff
@@ -52,8 +52,13 @@ class SetControllerViewModel(
         .filterIsInstance<StakingState.Stash>()
         .share()
 
-    val _isNotStashAccount = MutableLiveData<Boolean>()
-    val isNotStashAccount: LiveData<Boolean> = _isNotStashAccount
+    val showNotStashAccountWarning = accountStakingFlow.map { stakingState ->
+        stakingState.accountAddress != stakingState.stashAddress
+    }.asLiveData()
+
+    val isContinueButtonAvailable = accountStakingFlow.map { stakingState ->
+        showNotStashAccountWarning.value != null && showNotStashAccountWarning.value!!.not()
```

Comment:
Does not look reliable: showNotStashAccountWarning value is only considered on accountStakingFlow change, which generally won't happen more than once. Also `stakingState`  does not participate in mapping.
``` kotlin
val isContinueButtonAvailable = showNotStashAccountWarning.map(Boolean::not)
````

Reply:
Done